flink-java/src/test/java/org/apache/flink/api/java/operator/DataSinkTest.java

```diff
@@ -311,7 +311,7 @@ public void testPojoSingleOrderFull() {
 			.sortLocalOutput("*", Order.ASCENDING);
 	}
 
-	@Test(expected = InvalidProgramException.class)
+	@Test
 	public void testArrayOrderFull() {
```

Comment:
That is not how you fix a test. When the behaviour of the program changes, then one should also adapt the test accordingly. E.g. checking that the order of the elements is correct.